AWS_SERVICES_SUPPORTED.md

```diff
@@ -1,264 +0,0 @@
-# AWS Services Supported
```

Comment:
If/when we end up removing this, be sure to remember the reference to it in the README.